packages/react-components/src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import './App.css'
+import MIMap from './components/MIMap/MIMap'
+
+function App() {
+    return (
+        <>
+            <h1>MapsIndoors React Components</h1>
+            <h2><code>&lt;MIMap /&gt;</code></h2>
+            <div style={{ marginLeft: '5vw', width: "90vw", height: "70vh", boxShadow: '0px 0px 20px #000' }}>
+                <MIMap
+                    apiKey="mapspeople3d"
+                    gmApiKey={import.meta.env.VITE_GOOGLE_MAPS_API_KEY}
+                    mapboxAccessToken={import.meta.env.VITE_MAPBOX_ACCESS_TOKEN}
+                    center={{ lat: 57.05825378109134, lng: 9.951020621279326 }}
+                    zoom={19.2}
```

Comment:
Is the zoom 19,2 just a hard coded value?

Reply:
As we discussed in the office, `App.jsx` serves as a test app for the `MIMap` component. The prop values here are for testing purposes.